### ISSUE TITLE
Attributes

### ISSUE THREAD

**Issue by ItsMias** [zabi94/NBTTooltip#50]
I would love to see attributes like attack damage to show up! This is really useful for me and was the main reason I installed this mod, still extremely useful though!
![image](https://github.com/zabi94/NBTTooltip/assets/126190882/1d349a6d-6842-4682-9927-0b2daf9a115e)

**Comment by zabi94** [comment]
I'm not sure of what you're suggesting.
Adding info like attack damage with an icon like in the picture you attached?

**Comment by ItsMias** [comment]
No, sorry for the unclear message, in the screenshot I attached below you can see the attributes of a sword, I would like to see those attributes show up in the NBT tooltip part as well, they have been hidden by setting the HideFlags to 7, I however would like to see them even though they are hidden, this would save me a ton of time as I don't have to un-hide and re-hide them!
![image](https://github.com/zabi94/NBTTooltip/assets/126190882/2a247cfa-d025-4bc2-a1a9-ab5a63279bf8)

**Comment by zabi94** [comment]
Ah, I see.
The thing is, I don't decide what's in the NBT, and adding data that does nothing to the existing nbt is definitely not a good practice, and out of the scope of this mod.
